src/axios-orders.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const instance = axios.create({
+    baseURL: 'https://blog-app-e7a85.firebaseio.com/'
```

Comment:
look at filename `axios-orders.js`
do not copy-paste files from other projects.
or you can give a file name appropriate as per the functionality.

Reply:
sir, i have changed the file name